ledger-service/http-json-cli/src/main/scala/com/daml/http/Config.scala

```diff
@@ -129,21 +131,23 @@ private[http] object JdbcConfig
       s"${indent}url -- JDBC connection URL,\n" +
       s"${indent}user -- database user name,\n" +
       s"${indent}password -- database user password,\n" +
-      s"${indent}createSchema -- boolean flag, if set to true, the process will re-create database schema and terminate immediately.\n" +
+      s"${indent}createSchema -- boolean flag, if set to true, the process will re-create database schema and terminate immediately. This is deprecated and replaced by start-mode, however if set it will always overrule start-mode.\n" +
+      s"${indent}start-mode -- option setting how the schema should be handled. Valid options are ${DbStartupMode.allConfigValues
```

Comment:
@S11001001 is there some logic behind where we use camelCase and where we use kebab-case?
It seems like currently the option names are all kebab-case but then in the option-value we use camelCase?

If it was just the start mode, I’d say switching to kebab-case would make sense since then we’re consistent everywhere but it’s not. The websocket config still uses camelCase.